doc/misc/specs.7.rst

```diff
@@ -132,6 +132,19 @@ against NEVRAs and file provides.
 ``<package-name-spec>`` is matched against NEVRAs only.
 
 
+Dependency expressions
+======================
+
+When passing a ``<package-spec>``, the user can provide additional restriction
+rules for matching the arguments. Basic version comparisons, such as
+``<package-spec> >= <version>``, can be used for this purpose.
```

Comment:
I would name all operators (=, >=, <=). That's to tackle another problem in which people write == because DNF4 interprets it as =.